packages/@okta/vuepress-site/guides/sign-into-mobile-app/next-steps/index.md

```diff
@@ -3,7 +3,8 @@ title: Next Steps
 ---
 You should now understand how to sign users in to your mobile applications using Okta. If you got stuck, post a question in our [Developer Forums](https://devforum.okta.com).
 
-<!-- Link to Protect Your API Endpoints guide -->
+To learn how to protect the API endpoints your mobile app calls, see [Protect Your API Endpoints](/guides/protect-your-api/).
```

Comment:
To learn how to protect the API endpoints that your mobile app calls, see [Protect Your API Endpoints](/guides/protect-your-api/).